modules/k8s/node.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/gruntwork-io/terratest/modules/retry"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	// The following line loads the gcp plugin which is required to authenticate against GKE clusters.
```

Comment:
Be sure to add link to the ticket from the description.

Reply:
Ok